src/dpp/channel.cpp

```diff
@@ -79,7 +79,8 @@ forum_tag &forum_tag::set_name(const std::string &name) {
 	return *this;
 }
 
-const uint8_t CHANNEL_TYPE_MASK = 0b0000000000001111;
+const uint16_t CHANNEL_TYPE_MASK = 0b0000000000001111;
+const uint16_t DEFAULT_FORUM_LAYOUT_MASK = 0b0000011000000000;
```

Comment:
should we define a third bit here? just in case discord adds another layout type